src/routes/v2/auth.js

```diff
@@ -115,7 +116,7 @@ class AuthRouter {
 
   getRouter() {
     const router = express.Router()
-
+    router.use(this.rateLimiter)
```

Comment:
Thoughts on using the rate limiter purely on the `/verify` endpoint, since this is the brute-forceable component? The others can't really be brute forced, outside of sending email otps repeatedly via the `/login` endpoint, and we should always allow `/logout` and `/whoami`

Reply:
no strong opinion here, i can do this also! only concern is if the testers say something about spam (cos if we don't restrict, any malicious actor can spam to ask for OTP).

lmk if you think this is important enough to fix or can we leave as is

Reply:
I think it's important to remove the limits on `logout` and `whoami` - the rest i agree though!